src/components/header/style.scss

```diff
@@ -15,6 +15,8 @@ $md-header-width: 720px;
   width: 100%;
   max-width: 100vw;
   z-index: 100;
+  position: unset;
```

Comment:
I'm confused. You say below that position unset breaks the mobile nav so only use it when not in mobile, but then you're using it here. Does it only break the `menu-container` or should this be removed?

Reply:
I think I untentionally left that comment in from a copy pasta, let me double check